manifests/o/OpenChemistry/Avogadro2/1.91.0/OpenChemistry.Avogadro2.locale.en-US.yaml

```
@@ -0,0 +1,20 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: OpenChemistry.Avogadro2
+PackageVersion: 1.91.0
+PackageLocale: en-US
+Publisher: http://openchemistry.org/
+PublisherUrl: https://www.openchemistry.org/
+PackageName: Avogadro 2
```

```suggestion
PackageName: Avogadro2
```
This should match Control Panel